woocommerce/woocommerce
# add: stripe tax to onboarding tax task

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

3 possible screens:

- Manual setup: no tax plugins recommended (e.g Afghanistan)
- WooCommerce Tax & Stripe Tax recommended (e.g UK, US)
- Stripe Tax recommended (e.g Singapore)

1. Setup WooCommerce with a country to show one of the above flows
2. All flows should work as expected. (Stripe Tax is the newly added one but the components for all of them saw some change)
3. Stripe tax flow should install the plugin, then bring you to WC Settings
4. WooCommerce tax flow should go through the WC Tax automated setup flow

<img width="1705" alt="image" src="https://github.com/user-attachments/assets/062d6338-c4fc-4a9f-b1d5-5172d1b33577">
<img width="1699" alt="image" src="https://github.com/user-attachments/assets/791803c1-b53c-45b0-9c94-7d6dc92b48ec">
<img width="1705" alt="image" src="https://github.com/user-attachments/assets/3cd176c9-8e0a-417d-85be-d3d523670dec">





### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
